client/SDL/sdl_webview.hpp

```diff
@@ -26,8 +26,9 @@ extern "C"
 {
 #endif
 
-	BOOL sdl_webview_get_aad_auth_code(freerdp* instance, const char* hostname, char** code,
-	                                   const char** client_id, const char** redirect_uri);
+	BOOL sdl_webview_get_avd_access_token(freerdp* instance, char** token);
```

Comment:
Maybe a good idea to unify these.
something like `GetAccessToken(freerdp* instance, int tokenType, char** token, ...);` might do the trick.